app/src/modules/files/routes/item.vue

```diff
@@ -349,11 +356,6 @@ function discardAndStay() {
 	confirmLeave.value = false;
 }
 
-function downloadFile() {
```

Comment:
Similar to the first comment, perhaps we can "repurpose" this as a computed `downloadFileURL` or `downloadURL` instead of the direct addTokenToURL usage in template? Totally get that how using a computed can feel like an extra step, but just thinking aloud from a code consistency perspective. I'll defer to @rijkvanzanten for this 😉

Reply:
Yeah this should be a computed instead of doing it in-template 👍🏻